doc/source/ray-air/package-ref.rst

```diff
@@ -0,0 +1,136 @@
+.. _air-api-ref:
+
+AI Runtime API Reference
+========================
+
+.. contents::
+    :local:
+
+Components
+----------
+
+Preprocessors
+~~~~~~~~~~~~~
+
+.. autoclass:: ray.ml.preprocessor.Preprocessor
+    :members:
+
+.. automodule:: ray.ml.preprocessors
+    :members:
+    :show-inheritance:
+
+
+.. _air-trainer-ref:
+
+Trainer
+~~~~~~~
+
+.. autoclass:: ray.ml.trainer.Trainer
+    :members:
+
+.. automodule:: ray.ml.train.integrations.xgboost
+    :members:
+    :show-inheritance:
+
+.. automodule:: ray.ml.train.integrations.lightgbm
+    :members:
+    :show-inheritance:
+
+.. automodule:: ray.ml.train.integrations.tensorflow
+    :members:
+    :show-inheritance:
+
+.. automodule:: ray.ml.train.integrations.torch
+    :members:
+    :show-inheritance:
+
+.. automodule:: ray.ml.train.integrations.huggingface
+    :members:
+    :show-inheritance:
+
+.. automodule:: ray.ml.train.integrations.sklearn
+    :members:
+    :show-inheritance:
+
+.. autoclass:: ray.ml.train.data_parallel_trainer.DataParallelTrainer
+    :members:
+    :show-inheritance:
+
+.. autoclass:: ray.ml.train.gbdt_trainer.GBDTTrainer
+    :members:
+    :show-inheritance:
+
+
+
+.. _air-tuner-ref:
+
+Tuner
+~~~~~
+
+.. autoclass:: ray.tune.tuner.Tuner
+    :members:
+
+.. automodule:: ray.tune.result_grid
+    :members:
+
+Predictors
+~~~~~~~~~~
+
+.. autoclass:: ray.ml.predictor.Predictor
+    :members:
+
+.. autoclass:: ray.ml.batch_predictor.BatchPredictor
+    :members:
+
+.. automodule:: ray.ml.predictors.integrations.xgboost
+    :members:
+    :show-inheritance:
+
+.. automodule:: ray.ml.predictors.integrations.lightgbm
+    :members:
+    :show-inheritance:
+
+.. automodule:: ray.ml.predictors.integrations.tensorflow
+    :members:
+    :show-inheritance:
+
+.. automodule:: ray.ml.predictors.integrations.torch
+    :members:
+    :show-inheritance:
+
+.. automodule:: ray.ml.predictors.integrations.sklearn
+    :members:
+    :show-inheritance:
+
+.. automodule:: ray.ml.predictors.integrations.huggingface
+    :members:
+    :show-inheritance:
+
+.. _air-serve-integration:
+
+Serving
+~~~~~~~
+
+.. autoclass:: ray.serve.model_wrappers.ModelWrapperDeployment
+
+.. autoclass:: ray.serve.model_wrappers.ModelWrapper
+
+.. _air-results-ref:
+
+Outputs
```

Comment:
Probably not for this PR (too low prio), but the anchors for Outputs and Configs don't seem to work as expected. The nav "hangs" at Serving when scrolling down.

![Screenshot 2022-05-05 at 09 12 35](https://user-images.githubusercontent.com/3462566/166877421-fc80c35f-3727-4c29-a877-6da803a35aa9.png)